<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test1Dialog</class>
 <widget class="QDialog" name="test1Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>冷却水机组</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errordetection.qrc">
    <normaloff>:/images/images/timg.jpeg</normaloff>:/images/images/timg.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 冷却水出水水压</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>   冷却水水流量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  冷却水出水温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="widget_5" native="true">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>冷却水机组控制温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>220</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QWidget" name="widget_6" native="true">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>冷却水压缩机排气温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>222</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="widget_4" native="true">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>   冷却水水温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="errordetection.qrc"/>
 </resources>
 <connections/>
</ui>
